ingestion/functions/parsing/estonia/estonia.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+# Fixed location, all cases are for Estonia and no coarser location is available in the source.
+_LOCATION = {
+    "country": "Estonia",
+    "geoResolution": "Country",
+    "name": "Estonia",
+    "geometry": {
+        "longitude": "25.7615268448868",
+        "latitude": "58.7783968568071"
+    }
+}
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYY-mm-dd HH:MM:SS format (i.e.: 2020-03-06 18:44:00), but the date filtering API
+    expects mm/dd/YYYYZ format.
+    """
+    date = datetime.strptime(raw_date, "%Y-%m-%d %H:%M:%S")
+    return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender.upper() == "M":
+        return "Male"
+    elif raw_gender.upper() == "N":
+        return "Female"
+    elif raw_gender.upper() == "U":
+        # Unknown
+        return None
+    else:
+        raise ValueError(f'Unhandled gender: {raw_gender}')
+
+
+def convert_demographics(gender: str, age: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        # Brackets are like "35-39" or "over 85".
```

Comment:
Might be better to be explicit about these two options, rather than using the `else` statement here. For instance, if they decide to change format or add something like "under 1".

Reply:
right, now raising a value error if we can't parse it.